src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.test.tsx

```diff
@@ -101,19 +89,7 @@ describe('<BrowserRegionField', () => {
       jest.restoreAllMocks();
     });
 
-    test('resets region field when close button is clicked', () => {
-      const regionInput = createChrLocationValues().stringValue;
-
-      wrapper
-        .find(Input)
-        .simulate('change', { target: { value: regionInput } });
-      wrapper.find('button[role="closeButton"]').simulate('click');
-
-      expect(wrapper.find(Input).props().value).toBe('');
-      expect(wrapper.props().toggleRegionFieldActive).toHaveBeenCalledWith(
-        false
-      );
-    });
+    // TODO: Test if the form is reset when clicked outside the form. Need to be able to mock useOutsideClick for this.
```

Comment:
Here and in the other place, would it be possible to call the callback function passed on to the `useOutsideClick` directly and check if it resets the form?

Reply:
I tried a few different approaches to get test this, but unfortunately all of them failed. I'd prefer to defer this to a later time rather than holding this PR back.

Reply:
I tried a few different approaches to get test this, but unfortunately all of them failed. I'd prefer to defer this to a later time rather than holding this PR back.